meshio/permas_io.py

```diff
@@ -38,9 +38,7 @@ def read(filename):
         if re.search('\$ELEMENT TYPE = TRIA3', line):
```

Comment:
Same way as we did the other day:
 * Create a dictionary that translates meshio types into `TRIA3`, `QUAD4` etc
 * Create the search query `'\$ELEMENT TYPE = %s' % permas_type`
 * Then, the only thing that different is the number of point references that need to be extracted (e.g., `-4`). Those can be put in the same dictionary actually, e.g.,
```python
{
  'triangle': (3, 'TRI3')
  # ...
}
```